src/clients/Client.cc

```diff
@@ -818,15 +818,15 @@ void Client::handleAdaptedBodyProducerAborted()
 void
 Client::handleAdaptationCompleted()
 {
-    debugs(11,5, HERE << "handleAdaptationCompleted");
+    debugs(11,5, "handleAdaptationCompleted");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.